Cargo.toml

```diff
@@ -18,6 +18,7 @@ members = [
     "nomos-utils",
     "nodes/nomos-node",
     "nodes/nomos-node-api",
+    "nodes/nomos-node-types",
```

Comment:
nit: I would call this `nomos-node-lib`, where we put everything than could be reused by other packages